dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -61,35 +65,39 @@
-     * @param hgvsTStop end of the variant in a contig according to HGVS
-     * @param hgvsTOrientation Orientation of the contig to the chromosome (1 for forward, -1 for reverse)
+     * @param locusType         Locus type
+     * @param chromosome        Chromosome name, can be null if the contig is not mapped to a chromosome
+     * @param chromosomeStart   Start coordinate of the variant in chromosome, null if the contig is not fully mapped
+     *                          to a chromosome
+     * @param chromosomeEnd     End coordinate of the variant in chromosome, null if the contig is not fully mapped
+     *                          to a chromosome
+     * @param hgvsCReference    reference allele from HGVS table, when mapped into a chromosome
+     * @param hgvsTReference    reference allele from HGVS table, when mapped into a contig
+     * @param alternate         alternate allele
+     * @param alleles           reference and alternates alleles as submitted to DbSNP
+     * @param hgvsCString       HGVS annotation, mapping to a chromosome
+     * @param hgvsCStart        start of the variant in a chromosome according to HGVS
+     * @param hgvsCStop         end of the variant in a chromosome according to HGVS
+     * @param hgvsCOrientation  Orientation of the snp to the chromosome according to HGVS (1 for forward, -1 for
+     *                          reverse)
+     * @param hgvsTString       HGVS annotation, mapping to a contig
+     * @param hgvsTStart        start of the variant in a contig according to HGVS
+     * @param hgvsTStop         end of the variant in a contig according to HGVS
+     * @param hgvsTOrientation  Orientation of the contig to the chromosome (1 for forward, -1 for reverse)
      */
-    public SubSnpCoreFields(long subSnpId, int subSnpOrientation, Long snpId, int snpOrientation,
-                            String contig, Long contigStart, Long contigEnd, int contigOrientation,
-                            String chromosome, Long chromosomeStart, Long chromosomeEnd,
-                            String hgvsCReference, String hgvsTReference, String alternate, String alleles,
-                            String hgvsCString, Long hgvsCStart, Long hgvsCStop, int hgvsCOrientation,
-                            String hgvsTString, Long hgvsTStart, Long hgvsTStop, int hgvsTOrientation) {
+    public SubSnpCoreFields(long subSnpId, int subSnpOrientation, Long snpId, int snpOrientation, String contig,
+                            Long contigStart, Long contigEnd, int contigOrientation, LocusType locusType,
+                            String chromosome, Long chromosomeStart, Long chromosomeEnd, String hgvsCReference,
+                            String hgvsTReference, String alternate, String alleles, String hgvsCString,
+                            Long hgvsCStart, Long hgvsCStop, int hgvsCOrientation, String hgvsTString, Long hgvsTStart,
+                            Long hgvsTStop, int hgvsTOrientation) {
```

Comment:
[This comment](https://github.com/EBIvariation/eva-tools/pull/35#discussion_r145374067) suggests changing the orientations too, which I think makes sense. Using enums in some cases and ints in others is a bit incoherent.